search-commons/src/main/java/no/unit/nva/search2/enums/PublicationStatus.java

```diff
@@ -0,0 +1,5 @@
+package no.unit.nva.search2.enums;
+
+public enum PublicationStatus {
+   NONE, PUBLISHED, PUBLISHED_METADATA, DELETED, UNPUBLISHED, DRAFT, DRAFT_METADATA, DRAFT_DELETED
```

Comment:
Where are these from? Shouldn't it be NEW, DRAFT, PUBLISHED_METADATA, PUBLISHED, DELETED, UNPUBLISHED, DRAFT_FOR_DELETION?

Reply:
indeed